### PR TITLE
v4.2.1 Release Candidate

### PR DESCRIPTION
**Pull request by ZedThree**
This is a bugfix release. See change log for more details

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -139,11 +142,11 @@ authors:
 
   - family-names: Wang
     given-names: Zhanhui
-version: 4.2.0
-date-released: 2018-10-16
+version: 4.2.1
+date-released: 2019-01-??
```
Comment (bendudson) [inline CITATION.cff]: day, just a reminder

**Comment by ZedThree** [comment]
I've got one last bug fix to go in!

Anybody know of anything else that needs fixing in v4.2.0 before this release?

**Comment by ZedThree** [comment]
Just #1515 to go in, and then I'll update release date and merge

**Comment by ZedThree** [comment]
Ok, ready to go!

**Comment by bendudson** [comment]
Thanks @ZedThree !

**Pull request merged**